src/pages/Homepage.tsx

```diff
@@ -1,5 +1,312 @@
+    { id: 2, data: 'Dây Chuyền Bạc Nữ' },
+    { id: 3, data: 'Lắc tay bạc nữ' },
+    { id: 4, data: 'Nhẫn đôi bạc' },
+  ];
+  return (
+    <div className="w-full z-10">
+      <img src={banner} className="mb-[15px]" alt="banner" />
+
+      <div className="max-w-[1250px] mx-auto mb-6">
+        <div className="w-full flex flex-col md:flex-row">
+          <div className="w-full md:w-[60%] p-[15px] flex">
+            <img
+              src={subBanner_1}
+              alt="banner"
+              className="cursor-pointer transition-all hover:-translate-y-[6px] duration-300"
+            />
+          </div>
+          <div className="w-full md:w-[40%] p-[15px] flex">
+            <img
+              src={subBanner_2}
+              alt="banner"
+              className="cursor-pointer transition-all hover:-translate-y-[6px] duration-300"
+            />
+          </div>
+        </div>
+        <div className="w-full flex flex-col md:flex-row">
+          <div className="w-full md:w-[40%] p-[15px] flex">
+            <img
+              src={subBanner_3}
+              alt="banner"
+              className="cursor-pointer transition-all hover:-translate-y-[6px] duration-300"
+            />
+          </div>
+          <div className="w-full md:w-[60%] p-[15px] flex">
+            <img
+              src={subBanner_4}
+              alt="banner"
+              className="cursor-pointer transition-all hover:-translate-y-[6px] duration-300"
+            />
+          </div>
+        </div>
+      </div>
+
+      <div className="max-w-[1240px] mx-auto my-8 z-10">
+        <h1 className="mx-[10px] font-bold uppercase text-4xl text-primary mb-4">
+          <span>{t('flash-sale')}</span> <span>40%</span>
+        </h1>
+        <Carousel
+          slidesToScroll={1}
+          slidesToShow={4}
+          dots={false}
+          responsive={responsive}
+        >
+          <div className=" p-[10px]">
+            <ProductItem />
+          </div>
+          <div className="p-[10px]">
+            <ProductItem />
+          </div>
+          <div className="p-[10px]">
+            <ProductItem />
+          </div>
+          <div className="p-[10px]">
+            <ProductItem />
+          </div>
+        </Carousel>
+      </div>
+
+      <div className="mt-0 mb-12 px-4 relative w-full">
+        <div className="flex flex-col md:flex-row w-full gap-3 mx-auto">
+          <iframe
+            className="w-full md:w-1/2 h-[355px]"
+            allowFullScreen
+            src="https://www.youtube.com/embed/Wm8to68EPRw?feature=oembed&amp;start&amp;end&amp;wmode=opaque&amp;loop=0&amp;controls=1&amp;mute=0&amp;rel=0&amp;modestbranding=0"
+          ></iframe>
+          <div className="w-full md:w-1/2">
+            <Carousel slidesToScroll={1} slidesToShow={1} dots={true}>
+              {sliderApi.map((item, index) => (
+                <img key={index} src={item} alt="slider" />
+              ))}
+            </Carousel>
+          </div>
```

Comment:
E fix cho 2 thành phần này bằng nhau luôn khi phóng to, thu nhỏ, responsive màn hình nha